dojo/jira_link/views.py

```diff
@@ -34,7 +34,7 @@ def webhook(request):
 
     if request.method == 'POST':
         parsed = json.loads(request.body.decode('utf-8'))
-        if 'issue' in list(parsed.keys()):
+        if parsed['webhookEvent'] == 'jira:issue_updated':
```

Comment:
Wouldn't this throw a KeyError exception if the key is not present in the payload? (I am on the phone, cannot check much)

Reply:
From the Atlassian documentation on the Jira webhook:

"At a high level, every callback contains the webhookEvent ID, the timestamp, and information about the entity associated with the event (for example, issue, project, board, and so on). Callbacks may have additional information, depending on the type of event associated with it. As an example, the structure of a callback for an issue-related event is described next."

We should never see a call back without a "webhookEvent" key

Reply:
But let me rework.  I can get this to pass with out an error.

if parsed.get('webhookEvent') == 'jira:issue_updated':

should work either way.  If the condition doesn't match or the key doesn't exist, it will fail the test.